.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    env:
+      GEOS_VERSION: "3.9.1"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        # os: [ubuntu-20.04, windows-2019, macos-10.15]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v1.12.0
+        env:
+          CIBW_BUILD: cp38-manylinux_x86_64
+          CIBW_ENVIRONMENT_LINUX: GEOS_VERSION=${{ env.GEOS_VERSION }} GEOS_INSTALL=${{ github.workspace }}/geos GEOS_BUILD=${{ github.workspace }}/geosbuild GEOS_CONFIG=${{ github.workspace }}/geos/bin/geos-config LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${{ github.workspace }}/geos/lib
+          CIBW_ENVIRONMENT_MACOS: GEOS_VERSION=${{ env.GEOS_VERSION }} GEOS_INSTALL=${{ github.workspace }}/geos GEOS_BUILD=${{ github.workspace }}/geosbuild GEOS_CONFIG=${{ github.workspace }}/geosbuild/geos-${{ env.GEOS_VERSION }}/build/bin/geos-config
+          CIBW_ENVIRONMENT_WINDOWS: GEOS_VERSION=${{ env.GEOS_VERSION }} GEOS_INSTALL=${{ github.workspace }}\geos-${{ env.GEOS_VERSION }} GEOS_LIBRARY_PATH=${{ github.workspace }}\\geos-${{ env.GEOS_VERSION }}\\lib GEOS_INCLUDE_PATH=${{ github.workspace }}\\geos-${{ env.GEOS_VERSION }}\\include
```

Comment:
You can (probably?) split these over a few lines to improve their readability, i.e.:
```
          CIBW_ENVIRONMENT_LINUX:
            GEOS_VERSION=${{ env.GEOS_VERSION }}
            GEOS_INSTALL=${{ github.workspace }}/geos
            GEOS_BUILD=${{ github.workspace }}/geosbuild
            GEOS_CONFIG=${{ github.workspace }}/geos/bin/geos-config
            LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${{ github.workspace }}/geos/lib
```
And while I see it's not currently running, `GEOS_CONFIG` in `CIBW_ENVIRONMENT_MACOS` will need to be updated. Does `GEOS_BUILD` need to be specified?

Reply:
Yes it works indeed. Thanks. I also enabled macos and windows, let's see what happens.